### PR TITLE
Prescription Engineering Goggles

### PR DESCRIPTION
**Pull request by SomeCabbage**
# Description
Adds prescription security glasses
they have a small chance to fall off your face when you slip. Small enough that it wont be a problem normally but just enough so that it'll be noticeable

Adds 2 units to SecTech
Adds to loadouts for 4 points

# Media
image

# Changelog

:cl: SomeCabbage
- add: Added prescription engineering goggles and put them into the department vending machine and loadout selection.

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   id: EngiVendInventory
   startingInventory:
     ClothingEyesGlassesMeson: 4
+    ClothingEyesGlassesMesonPrescription: 4
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/engivend.yml]:
```suggestion
    ClothingEyesGlassesMesonPrescription: 2
```

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@
     Flash: 5
     FlashlightSeclite: 5
     ClothingEyesGlassesSecurity: 4
+    ClothingEyesGlassesSecurityPrescription: 4
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/sec.yml]:
```suggestion
    ClothingEyesGlassesSecurityPrescription: 2
```

**Comment by Pspritechologist** [comment]
Cabbage.
I demand you explain to me the fundamental operation of a *prescription hud* and how that makes you see better.

**Comment by SomeCabbage** [comment]
At most it would let you see better out of one eye since the screen/glass/hud needs to be within your field of vision, but other than that I have no idea.

**Pull request closed without merging**